server/src/lsp4clj/core.clj

```diff
@@ -69,6 +69,19 @@
 
 (set! *warn-on-reflection* true)
 
+(def null-output-stream-writer
+  (java.io.OutputStreamWriter.
+    (proxy [java.io.OutputStream] []
+      (write
+        ([^bytes b])
+        ([^bytes b, off, len])))))
```

Comment:
Usually when using GraalVM we avoid adding top-level defs as it makes graal build slower, but this is a simple class instantiation so I guess it's ok.

Reply:
Hmm... I didn't know that. Do you know why that slows it down? Anyway, I'll leave this as is unless you say otherwise.